src/migrations/scripts/1597928323618-initial.ts

```diff
@@ -83,6 +83,44 @@ function migrationCommands (transaction: any): Commands {
         }
       ]
     },
+    {
+      fn: 'createTable',
```

Comment:
Uff I would say this should be a new migrations. The idea of migrations are that they are immutable and don't change, but only add-on. Yes it was not released yet, actually it was already as the pre-releases, so yes this should be a new migration...

Reply:
Sure